test/commands/break_test.rb

```diff
@@ -19,31 +19,36 @@ def program
         12:      end
         13:    end
         14:
-        15:    y = 3
-        16:
-        17:    byebug
+        15:    def self.#{example_module_method}
```

Comment:
Can't you just call this `self.c` (for example) and get rid of the `example_module_method`? The `example_class` method is there to prevent some redefinition warnings, but I don't think something like that is necessary here.